gapic/src/main/com/google/gapid/perfetto/views/TraceConfigDialog.java

```diff
@@ -180,6 +183,12 @@ public static String getConfigSummary(Settings settings, Device.PerfettoCapabili
     PerfettoConfig.TraceConfig.Builder config = PerfettoConfig.TraceConfig.newBuilder();
     SettingsProto.PerfettoOrBuilder p = settings.perfetto();
 
+    PerfettoConfig.FtraceConfig.Builder ftrace = config.addDataSourcesBuilder()
+          .getConfigBuilder()
+              .setName("linux.ftrace")
+              .getFtraceConfigBuilder()
+                  .setBufferSizeKb(FTRACE_BUFFER_SIZE);
+
```

Comment:
This causes the ftrace datasource to be added even if we aren't collecting cpu or gpu data. This should be wrapped in an `if (p.getCpuOrBuilder().getEnabled() || (p.getGpuOrBuilder().getEnabled())`